import-scripts/monitor-crontab-version.sh

```diff
@@ -14,20 +14,30 @@ if [ -z $PORTAL_HOME ] | [ -z $PIPELINES_CONFIG_HOME ] ; then
     exit 1
 fi
 
-GITHUB_CRONTAB_URL="https://api.github.com/repos/knowledgesystems/cmo-pipelines/contents/import-scripts/mycrontab"
+GITHUB_PIPELINES_CRONTAB_URL="https://api.github.com/repos/knowledgesystems/cmo-pipelines/contents/import-scripts/pipelines/mycrontab"
+GITHUB_PIPELINES_EKS_CRONTAB_URL="https://api.github.com/repos/knowledgesystems/cmo-pipelines/contents/import-scripts/pipelines_eks/mycrontab"
 GITHUB_AUTHORIZATION=$(cat $PIPELINES_CONFIG_HOME/git/git-credentials | sed 's/https:\/\///; s/@github\.com//')
 CURRENT_CRONTAB_FILE="$PORTAL_HOME/tmp/current_crontab"
 GITHUB_CRONTAB_FILE="$PORTAL_HOME/tmp/git_crontab"
 REQUIRED_USERNAME="cbioportal_importer"
-REQUIRED_HOSTNAME="pipelines.cbioportal.mskcc.org"
+PIPELINES_HOSTNAME="pipelines.cbioportal.mskcc.org"
+PIPELINES_EKS_HOSTNAME="ip-10-1-21-42.mskcc.root.mskcc.org"
```

Comment:
maybe we should create a symbolic hostname / dns record for the eks pipelines node?  Otherwise this code will break if we ever redeploy the eks pipelines node and it is assigned a new IP address.